client-app/shared/account/components/account-navigation.vue

```diff
@@ -100,6 +93,13 @@
         :text="$t('shared.account.navigation.links.notifications')"
         class="notifications-icon"
       />
+
+      <AccountNavigationLink
```

Comment:
If I'm not mistaken, you wanted the main app to know as less as possible about modules (ideally nothing)
maybe account-navigation can be extended by Teleport here?

Reply:
Why not. But it will not be always in the DOM
I dont see how to do it. Let me know

Reply:
I'll try to figure it out a bit later, will let you know if I'll get it :)